INSTALL.md

```diff
@@ -110,7 +108,7 @@ Example of calling cmake with options
 
 ```sh
 cmake \
-  -D ENABLE_STATIC=OFF \
+  -D BUILD_SHARED_LIBS=OFF \
```

Comment:
I don't know the context here, but "ENABLE_STATIC=OFF" -> "BUILD_SHARED_LIBS=OFF" sounds odd.